src/config.js

```diff
@@ -25,18 +25,20 @@ const defaultConfig = {
   "snippetAnonDefault": false,
 
   "status": "DM me to contact staff!",
-  "openingMessage": "Hi there, thanks for reaching out to the Dyno modmail bot. This is the best way to get in contact with the Dyno staff team. Please select what you'd like assistance with from the options below so I can route your request to the correct team.\n\n*Note: We can only offer assistance in English.*",
+  "openingMessage": "Hi there, thanks for reaching out to the Dyno modmail bot.\n This is the best way to get in contact with the Dyno staff team. Please select what you'd like assistance with from the options below so I can route your request to the correct team.\n\n*Note: We can only offer assistance in English.*",
```

Comment:
```suggestion
  "openingMessage": "Hi there, thanks for reaching out to the Dyno modmail bot.\nThis is the best way to get in contact with the Dyno staff team. Please select what you'd like assistance with from the options below so I can route your request to the correct team.\n\n*Note: We can only offer assistance in English.*",
```

Reply:
This is intended so that it lines up better with the emoji that will be on the first line of the embed.